watertap/core/membrane_channel_base.py

```diff
@@ -233,6 +233,20 @@ class FrictionFactor(Enum):
     ),
 )
 
+CONFIG_Template.declare(
+    "transport_model",
+    ConfigValue(
+        default="soultion-diffusion",
```

Comment:
```suggestion
        default="solution-diffusion",
```
I just wondered how the tests would pass if this typo were here, but then I realized that you need to add a couple of new tests for 0DRO, 1DRO, and probably OARO 0D to check that the formulation gets you to expected results.

Reply:
Additionally, I think you need to add an enumerated class for transport_model and include the `solution_diffusion` and `SKK` options there. Then you would add that class in here as the domain and also describe SKK in docstring.

Reply:
Addressed in commit c6bc666